protobufs/generated/messages/events.proto

```diff
@@ -266,6 +267,7 @@ enum EventType {
   // DIAGNOSE_COMMAND__POST_ENTER_IN_ERROR = 14009;
   // DIAGNOSE_COMMAND__PRE_EXECUTE_ACTION = 14010;
   // DIAGNOSE_COMMAND__POST_EXECUTE_ACTION = 14011;
+  // DIAGNOSE__DIAGNOSE__POST_SEARCH = 14012;
```

Comment:
Is this refactored command not available yet? This is a particularly high-value one.

Reply:
whenever i try to use it, i'm still getting an error that its missing an adapter

Reply:
You are correct! Sorry, I thought that command had been merged.